src/OnnxParser/nodes/linear.jl

```diff
@@ -264,6 +280,8 @@ end
 
 onnx_node_to_flux_layer(node::ONNXConvT) = node.convt
 
+islinear(node::ONNXConvT) = true
```

Comment:
Regarding convolution:
Is your plan to convert these first into a Matrix and then contract the matrix or to introduce a special treatment for them later on? Just wondering whether that's something we already need to worry about at the`islinear` level, but I of course agree that convolution is linear :)